CONTAINERS.md

```diff
@@ -35,17 +35,33 @@ It also runs database migrations.
 We also provide a base image which can be reused for Hyrax applications: `hyrax`.
 
 ```sh
-echo "FROM hyrax" > Dockerfile
+echo "FROM samveralabs/hyrax" > Dockerfile
```

Comment:
As someone Docker naive, from where would I run this? I don't think I'd want to run it in the root of my hyrax git repository.

Reply:
in the root of your own hyrax application.

maybe the context of this isn't clear?

Reply:
It's not immediately clear, but it's not inscrutable.